Assets/UI/worldinput_CQUI.lua

```diff
@@ -246,6 +246,23 @@ function DefaultKeyUpHandler( uiKey:number )
         cquiHandledKey = true;
     end
 
+    if action["SPREAD_RELIGION"] and CQUI_UnitSupportsSpreadingReligion(unitType) then
+        UnitManager.RequestOperation(selectedUnit, GameInfo.UnitOperations["UNITOPERATION_SPREAD_RELIGION"].Hash);
+        cquiHandledKey = true;
+    end
```

Comment:
I think having a operation based check would be more future proof, in case they decide to add other units that can do the same operation.

```suggestion
    local bCanSpreadReligion = selectedUnit and UnitManager.CanStartOperation(selectedUnit, GameInfo.UnitOperations["UNITOPERATION_SPREAD_RELIGION"].Hash, nil, true);
    if action["SPREAD_RELIGION"] and bCanSpreadReligion then
        UnitManager.RequestOperation(selectedUnit, GameInfo.UnitOperations["UNITOPERATION_SPREAD_RELIGION"].Hash);
        cquiHandledKey = true;
    end
```

Reply:
thank you, I was not aware of that "can start operation" function.